elyra/validation/__init__.py

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
We shouldn't need to add a new folder, this is all in the context of Pipelines so it should be on the pipeline folder.